README.rst

```diff
@@ -5,23 +5,21 @@ Ansible
 *******
 
 Ansible is a radically simple IT automation system. It handles
-configuration-management, application deployment, cloud provisioning,
-ad-hoc task-execution, and multinode orchestration -- including
-trivializing things like zero-downtime rolling updates with load
-balancers.
+configuration management, application deployment, cloud provisioning,
+ad-hoc task execution, and multi-node orchestration. Ansible makes complex
+changes like zero-downtime rolling updates with load balancers easy. More information on `the Ansible website <https://ansible.com/>`_.
```

Comment:
'rolling updates with load balancers' seems like a very specific use case, but maybe it's the best example

Reply:
I think this is OK. It's a good example of something that's more complex.